macros/recursive_dag.sql

```diff
@@ -59,7 +59,7 @@ all_relationships (
         directory_path as child_directory_path,
         file_name as child_file_name,
         0 as distance,
-        {{ dbt_utils.array_construct(['resource_name']) }} as path,
+        {{ array_construct(['resource_name']) }} as path,
```

Comment:
And then these should be `dbt_project_evaluator.array_construct()` for now and swap out to `dbt.array_construct()` after 1.3

Reply:
unqualified name should also work here, correct?

Reply:
i.e.`{{ array_append() }}`

Reply:
( i updated to follow your suggestino, but both seem to work and just confirming my understanding)

Reply:
Uhhhhh in my experience, macros inside of a package have always had to be qualified. Were you installing this package into another project and then calling it, or just testing from inside of itself?

Reply:
testing from within the nested `integrations_test` subdirectory